tests/unit/PolicyTest.js

```diff
@@ -1,104 +1,65 @@
 import * as Policy from '../../src/libs/actions/Policy';
 
```

Comment:
can you not just create a variable here called `allPolicies` that would have the policy details that it would grab from? Haven't tried this yet so not sure though

Reply:
This wouldn't work. I went ahead and added a second parameter to the method, which is passed down from the tests

Reply:
There should be a way to mock local variables in jest from this brief [google search](https://www.google.com/search?q=jest+mock+local+variable&oq=jest+mock+local+variable&aqs=chrome.0.69i59j69i57l2j69i60l2j69i64l3.2852j0j4&sourceid=chrome&ie=UTF-8) - maybe setting this.allPolicies or window.allPolicies?

We don't want to adjust the code just so that we can pass testing parameters through it